docs/site/Relations.md

```diff
@@ -0,0 +1,25 @@
+---
+lang: en
+title: 'Relations'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Relations.html
+summary:
+---
+
+## Overview
+
+Model relation in LoopBack 3 is one of its powerful features which help users
```

Comment:
Please consider adding more content from https://loopback.io/doc/en/lb3/Creating-model-relations.html

Reply:
I thought the content there is still relevant so I've copied some stuff from the overview.